glossary/README.md

```diff
@@ -0,0 +1,20 @@
+# QF Project Glossary
+
+This folder contains the comprehensive glossary for QF Project, organized into three separate markdown files for easy reference.
+
+## Files in this Folder
+
+### 1. [glossary.md](./glossary.md)
+**Glossary - Actively Used in Project**
+
+Contains terms that are actively implemented and used within the QF Project blockchain ecosystem. These terms represent the core components, technologies, and concepts that make up the current QF Project implementation, including consensus mechanisms like SPIN, node types, infrastructure components, and user interfaces. This is the primary reference for understanding the technical foundation of QF Project.
+
+### 2. [glossary-RnD process.md](./glossary-RnD process.md)
```

Comment:
Broken link rendering for a file with a whitespace in the name. Let's use `-` instead of the whitespace? Also if using kebeb-case could be `rnd,` not `RnD`.